src/profiler.cpp

```diff
@@ -74,6 +74,7 @@ enum EventMask {
     EM_LOCK  = 4,
     EM_WALL  = 8,
     EM_NATIVEMEM = 16,
+    EM_INSTRUMENT = 32,
```

Comment:
The comment above says these constants are the same we use in `JfrSync`. Is this correspondence something we want to keep @apangin ?

Reply:
Why not?

Reply:
This was settled in #1507